cmd/dotenv/template.go

```diff
@@ -123,28 +138,53 @@ func writeUsingTemplateFile(dotenvFile string) ([]variable, string, string, erro
 }
 
 func writeContents(contents, dotenvFile, templateFile string) error {
+	// Strip any existing header comments from the contents
+	lines := strings.Split(contents, "\n")
+
+	filteredLines := make([]string, 0, len(lines))
+
+	strippedHeader := false
+
+	for _, line := range lines {
+		trimmed := strings.TrimSpace(line)
+		// Skip lines that are our generated headers (at the start of the file)
+		if !strippedHeader && (strings.HasPrefix(trimmed, "# Autogenerated by dr dotenv") ||
```

Comment:
Kind of a nit but feels like the `# Autogenerated by dr dotenv` magic string should be a constant.

And reused elsewhere - so we have one canonical place where the actual string lives.

Reply:
Good call, yeah